sunpy/io/file_tools.py

```diff
@@ -53,13 +54,13 @@ def __getitem__(self, key):
 })
 
 
-def read_file(filepath, filetype=None, **kwargs):
+def read_file(filename, filetype=None, **kwargs):
     """
     Automatically determine the filetype and read the file.
 
     Parameters
     ----------
-    filepath : `str`
+    filename : `str` or file-like
```

Comment:
Is this still not a filepath?

Reply:
It was just a rename, to distinguish between the arg and the path extracted from IO object